### PR TITLE
Add support for comms in LFG/LFD.

### LINKED ISSUE #5165: "You aren't in a party" spam
## Bug description

After the latest update of WotLK classic I have few system messages "You aren't in a party" when I enter the dungeon through the LFG. I tried with only one Questie enabled and still see this chat spam.

## Questie & WoW version

Wotlk Classic 3.4.3
Questie v8.10.6

### PR DESCRIPTION
**Pull request by bjthompson805**
## Issue references

Fixes #5165

## Proposed changes

- Add support for broadcasting when in an instance group, such as LFD.

I tested this in-game and it no longer shows the error mentioned in the above issue. I also confirmed that it is sharing data with other members of the instance group. I made it send as BULK in case BG messages are ever enabled.

### CODE REVIEW

**Comment by BreakBB** [comment]
Hey @bjthompson805 awesome that you took the time to look into this issue and provide a fix! We appreciate it <3

I am looking into fixing the pipeline (nothing serious, just luacheck not knowing about `LE_PARTY_CATEGORY_INSTANCE`) and we plan to ship this in a release later today.

File: Modules/QuestiePlayer.lua [hunk 1]
```diff
@@ -72,6 +72,8 @@ end
 function QuestiePlayer:GetGroupType()
     if(UnitInRaid("player")) then
         return "raid";
+    elseif(IsInGroup(LE_PARTY_CATEGORY_INSTANCE)) then
```
Comment (BreakBB) [inline Modules/QuestiePlayer.lua:75]:
```suggestion
    elseif(IsInGroup(Enum.ChatChannelType.PublicParty)) then
```

**Comment by bjthompson805** [comment]
@BreakBB The enum you added doesn't work. The value is 3, but the value of the constant I was trying to use (LE_PARTY_CATEGORY_INSTANCE) is 2. The error is gone but comms in instance groups isn't working.

**Comment by MyTechnoHunter** [comment]
Dump: value=Enum.ChatChannelType
[1]={
  Custom=1,
  PrivateParty=2,
  Communities=4,
  PublicParty=3,
  None=0
}

thaws the value from the game..

this is the value from the blizzard interface code..
image


the defined value is literally a copy of the definition?

**Comment by MyTechnoHunter** [comment]
jump in the discord please :) faster to talk there

**Comment by MyTechnoHunter** [comment]
testing in game verifies your statement, /run print(LE_PARTY_CATEGORY_INSTANCE) indeed returns 2...

-ponder- dont blame BreakBB for this one, its on me..

**Comment by MyTechnoHunter** [comment]
https://github.com/wowless/wowless/blob/fa9f8e7f4cb0f5acdb985f24dc9dc219ce2c9744/data/products/wow_classic/globals.yaml#L6010

link to the C-Code define of the LE_ proper.. oops?

**Comment by bjthompson805** [comment]
@MyTechnoHunter Your screenshot is of classic-era, maybe that's why?

@BreakBB also released a bug today with the dungeon daily thing... it's super broken, I had to disable questie to be able to accept the 2nd quest from the quest giver. Can we just fix both of these?

At least the error is gone lol. :)

**Comment by MyTechnoHunter** [comment]
the screenshot was from earlier today :) its the same in the wrath folder for Blizz code..
turns out the LE_ are defined in C code in the game client not our lua code.. there appears to only be LE_PARTY_CATEGORY_ HOME and INSTANCE

-shrug- 

got more info about the dailies being broken?

**Comment by bjthompson805** [comment]
Hm, it doesn't seem to be happening anymore. It may have been another addon and when it reloaded the UI, it fixed it.

Can probably ignore it.

**Comment by MyTechnoHunter** [comment]
aight

**Pull request merged**